Packs/ApiModules/Scripts/FormatURLApiModule/FormatURLApiModule.py

```diff
@@ -935,7 +935,7 @@ def remove_trailing_bracket_and_comma_from_part(part: str, inside_brackets: int)
     """
     Removes trailing bracket and commas from a part of a URL.
     """
-    if part.endswith(",") and inside_brackets:
+    if part.endswith((",", ")", "]")) and inside_brackets:
```

Comment:
This will not work, it will break the formatter.
`https://test.com/abc?q=(123)` -> `https://test.com/abc?q=(123`